packages/ag-charts-website/src/content/docs/charts-radial-column-series/index.mdoc

```diff
@@ -0,0 +1,186 @@
+---
+title: "Radial Column Series"
+enterprise: true
+---
+
+**Radial Column** series (sometimes called *Circular Column chart*) represents a dataset in the form of rectangular columns placed along the polar axis. It is similar to [Nightingale Chart](./charts-nightingale-series).
+
+## Grouped Radial Column
+
+To create a basic Radial Column chart, use a data array where each item has one category value (for the angle axis) and one numeric value (for the radius axis).
+
+A minimal Radial Column series configuration will look like:
+
+```js
+series: [{
+    type: 'radial-column',
+    angleKey: 'product',
+    radiusKey: 'sales',
+}]
+```
+
+{% chartExampleRunner title="Grouped Radial Column Chart" name="group-radial-column" type="generated" options="{ \"enterprise\": true }" /%}
+
+## Stacked Radial Column
+
+In a stacked Radial Column series, columns are placed on top of each other to create a total for each category. The following example illustrates a stacked Radial Column chart:
+
+```js
+series: [
+    {
+        type: 'radial-column',
+        angleKey: 'product',
+        radiusKey: 'A sales',
+        stacked: true,
+    },
+    {
+        type: 'radial-column',
+        angleKey: 'product',
+        radiusKey: 'B sales',
+        stacked: true,
+    },
+],
+```
+
+{% chartExampleRunner title="Stacked Radial Column Chart" name="stack-radial-column" type="generated" options="{ \"enterprise\": true }" /%}
+
+## Additional Styling Options
+
+There is a way to set a padding between columns. To do this, a few options should be added to the polar **angle** axis. `paddingInner` controls the padding between column groups, and `groupPaddingInner` controls the padding between columns within a group.
```

Comment:
The column spacing in the Radial Column chart can be adjusted by modifying the polar ••angle•• axis configuration. Use `paddingInner` to control gaps between column groups, and `groupPaddingInner` to manage gaps between individual columns within groups.